sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc

```diff
@@ -192,11 +200,15 @@ It must be the result of a constant instruction. +
  +
 'Scope' is memory scope for operations on the matrix. It must be the
 result of a constant instruction with scalar 'integer type'. +
+ +
+'Use' is an optional parameter that shows where in a math operation the matrix
+is used. It must be the result of a constant instruction with scalar
+'integer type'. +
 
 
 1+|Capability: +
 *{capability_name}*
-1+| 7 | {OpTypeJointMatrixINTEL_token}
+1+| 8 | {OpTypeJointMatrixINTEL_token}
```

Comment:
`Use` is an optional argument, which means that operands count is not always `8`, it should probably say `7+` instead